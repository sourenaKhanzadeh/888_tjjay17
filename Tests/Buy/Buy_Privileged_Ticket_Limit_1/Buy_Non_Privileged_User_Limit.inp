login
buy-standard
buy
Valid Title
6
logout