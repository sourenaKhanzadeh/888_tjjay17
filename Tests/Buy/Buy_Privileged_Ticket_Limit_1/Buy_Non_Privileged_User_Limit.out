Welcome to the system
Enter session type
Enter operation
Enter the event title you would like to buy
Enter the number of tickets you would like to purchase
The maximum limit of tickets for purchase is 4, please try again.
Session terminated
